walker_lake/walker_lake.ipynb

```diff
@@ -200,25 +209,21 @@
     "# Resampling to same size"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The two images share the same coordinate system but do not have the exact same dimensions or coordinates. Previous versions of this notebook resampled the images to the same size, and optionally allowed to regrid them, all using  Datashader. In this version, we now interpolate the Landsat-8 image to fit onto the coordinates of the Landsat-5 one using `xarray`, approach that provides a similar result."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import datashader as ds\n",
-    "from datashader import reductions as rd\n",
-    "\n",
-    "x_range = (int(max(ndvi5.x.min(), ndvi8.x.min())), int(min(ndvi5.x.max(), ndvi8.x.max())))\n",
-    "y_range = (int(max(ndvi5.y.min(), ndvi8.y.min())), int(min(ndvi5.y.max(), ndvi8.y.max())))\n",
-    "\n",
-    "plot_width = min(ndvi8.dims['x'], ndvi5.dims['x'])\n",
-    "plot_height = min(ndvi8.dims['y'], ndvi5.dims['y'])\n",
-    "cvs = ds.Canvas(plot_width=plot_width, plot_height=plot_height,\n",
-    "                x_range = x_range, y_range = y_range)\n",
-    "\n",
-    "ndvi5 = cvs.raster(ndvi5, agg= rd.mean('ndvi'))\n",
-    "ndvi8 = cvs.raster(ndvi8, agg= rd.mean('ndvi'))"
+    "ndvi8 = ndvi8.interp_like(ndvi5, method=\"nearest\")\n",
```

Comment:
Very nice! I am assuming the result is very close to the old method.

Reply:
Yes, it is. I also compared it with the output of `rioxarray.project_match` and there was no difference.